### PR TITLE
Limit logs websocket to `strict` auth mode

### PR DESCRIPTION
**Pull request by br41nslug**
Some tweaks for https://github.com/directus/directus/pull/23016

## Scope

What's changed:

A quick attempt at implementing my own suggestions

- Limit to `strict` auth mode
- Auth timeout set to `0` to disconnect when it expires
- Fixed endpoint to `/websocket/logs`

### CODE REVIEW

File: api/src/websocket/handlers/logs.ts [hunk 1]
```diff
@@ -22,86 +17,11 @@ export class LogsHandler {
 
 		this.controller = controller;
 		this.messenger = useBus();
-		this.subscriptions = new Set();
```
Comment (licitdev) [inline api/src/websocket/handlers/logs.ts:25]: I think we should not remove this to filter by log levels in different streams.

**Review by licitdev — COMMENTED** [review]
Yes, `strict` sounds good for this websocket endpoint. Should a long standing connection be required, a static token can be used.

The admin check has been removed and should be re-added. 

I prefer the existing code structure instead of the removal, as the functionality still needs to be extended. It is also consistent with your existing implementation for the WS subscription.

**Comment by br41nslug** [comment]
> I prefer the existing code structure instead of the removal, as the functionality still needs to be extended. It is also consistent with your existing implementation for the WS subscription.

Up to you 😄 Thats why I made this a separate PR instead of just pushing the changes 🚀

**Review by licitdev — APPROVED** [review]
@br41nslug Thank you! 🫡

**Pull request merged**